.gitignore

```diff
@@ -5,4 +5,6 @@ __pycache__/
 README.md
 setup.py
 scratchattach/test.py
-scratchattach.code-workspace
\ No newline at end of file
+scratchattach.code-workspace
+.DS_Store
+wiki/.DS_Store
```

Comment:
wiki/.DS_Store is not needed.
_I'm starting to think every contributor to this project doesn't know basic repository management_

Reply:
actually, I am not sure that's true, but you can just replace both with "**/.DS_STORE"